config/locales/en.yml

```diff
@@ -2441,13 +2441,15 @@ en:
     directions:
       ascend: "Ascend"
       engines:
+        fossgis_osrm_bike: "Bicycle (FOSSGIS OSRM)"
+        fossgis_osrm_car: "Car (FOSSGIS OSRM))"
+        fossgis_osrm_foot: "Foot (FOSSGIS OSRM))"
```

Comment:
We seem to have a superfluity of parentheses here...

Reply:
the parentheses should go, as to the naming: no strong feelings historically we've indicated the routing engine in one form or the other

Reply:
Changed to ..... (OSRM) as we have attribution of FOSGISS in the footer of the routing results.